lib/generators/manageiq/plugin/templates/README.md

```diff
@@ -1,9 +1,8 @@
 # <%= class_name %>
 
-[![Gem Version](https://badge.fury.io/rb/<%= plugin_name %>.svg)](http://badge.fury.io/rb/<%= plugin_name %>)
-[![Build Status](https://travis-ci.org/ManageIQ/<%= plugin_name %>.svg?branch=master)](https://travis-ci.org/ManageIQ/<%= plugin_name %>)
-[![Code Climate](https://codeclimate.com/github/ManageIQ/<%= plugin_name %>.svg)](https://codeclimate.com/github/ManageIQ/<%= plugin_name %>)
-[![Test Coverage](https://codeclimate.com/github/ManageIQ/<%= plugin_name %>/badges/coverage.svg)](https://codeclimate.com/github/ManageIQ/<%= plugin_name %>/coverage)
+[![Build Status](https://travis-ci.com/ManageIQ/<%= plugin_name %>.svg?branch=master)](https://travis-ci.com/ManageIQ/<%= plugin_name %>)
+[![Maintainability](https://api.codeclimate.com/v1/badges/<badge_token>/maintainability)](https://codeclimate.com/github/ManageIQ/<%= plugin_name %>/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/<badge_token>/test_coverage)](https://codeclimate.com/github/ManageIQ/<%= plugin_name %>/test_coverage)
```

Comment:
Unfortunately, we have a chicken-egg problem here with the badge token...this will just have to be an after-the-fact thing.